MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSCrashesUtil.m

```diff
@@ -5,6 +5,7 @@
 
 static NSString *const kMSCrashesDirectory = @"com.microsoft.azure.mobile.mobilecenter/crashes";
 static NSString *const kMSLogBufferDirectory = @"com.microsoft.azure.mobile.mobilecenter/crasheslogbuffer";
+static NSString *const kMSWrapperExceptionsDirectory = @"com.microsoft.azure.mobile.mobilecenter/crasheswrapperexceptions";
```

Comment:
It would be better to have a base directory as a constant for "com.microsoft.azure.mobile.mobilecenter".